unlock-app/package.json

```diff
@@ -74,8 +74,9 @@
     "deploy-staging": "npm run build && TARGET=staging npm run build-unlockjs && aws s3 sync build/ s3://staging.unlock-protocol.com --profile unlock --no-progress | awk '{print $2;}' | cut -d'/' -f2- | awk '{print \"/\"$0}' | xargs aws cloudfront create-invalidation --distribution-id E1ZWU0E90B9EBN --paths --profile unlock",
     "build-unlockjs": "./node_modules/webpack/bin/webpack.js --config webpack-unlockjs.config.js",
     "serve-unlockjs": "TARGET=dev node_modules/.bin/webpack-dev-server --config webpack-unlockjs.config.js &",
-    "test": "NODE_ENV=test jest --env=jsdom",
+    "test": "cd ../smart-contracts && truffle compile && cd ../unlock-app && NODE_ENV=test jest --env=jsdom",
```

Comment:
Please do not change this test script.
If you need, use `pretest` in another PR.